src/java.naming/share/classes/com/sun/jndi/ldap/VersionHelper.java

```diff
@@ -59,7 +59,7 @@ public final class VersionHelper {
         // System property to control whether classes is allowed to be loaded from
         // 'javaSerializedData' attribute
```

Comment:
Hello Aleksei, should we also update this code comment to make a mention of the additional attributes this system property now controls?

Reply:
Hi Jai,
Thanks for spotting that - will update this and other code comments in the `VersionHelper` class.